etc/picongpu/summit-ornl/gpu_picongpu.profile.example

```diff
@@ -33,20 +33,20 @@ module load boost/1.66.0
 module load hdf5/1.10.3
 module load adios/1.13.1-py2 c-blosc zfp sz lz4
 
-# download libSplash and compile it yourself from
+# optionally download libSplash and compile it yourself from
 #   https://github.com/ComputationalRadiationPhysics/libSplash/
-export Splash_DIR=$PROJWORK/$proj/huebl/sw/splash-1.7.0
-export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$Splash_DIR/lib
+# export Splash_ROOT=<your libSplash install directory, e.g., ${HOME}/sw/libSplash-1.7.0>
```

Comment:
```suggestion
# export Splash_ROOT=<your libSplash install directory> e.g., ${HOME}/sw/libSplash-1.7.0>
```
But I am not sure the in-line ` e.g. more text` will help users that just comment-in the line...
maybe
```suggestion
# export Splash_ROOT=<your libSplash install directory>  # e.g., ${HOME}/sw/libSplash-1.7.0>
```
instead

Reply:
Okay, I will follow the latter option.